RELEASE_NOTES.md

```diff
@@ -2,11 +2,12 @@
 
 ## Summary
 
-<!-- Here goes a general summary of what this release is about -->
+This update introduces the possibility to target specific types of batteries, inverters and EV chargers.
+
 
 ## Upgrading
 
-* The `TargetComponents` message now accepts an optional `type` too. `.components.component_categories` is no longer just a `ComponentCategory` but a new `CategoryAndType` message that has a required `category` (`ComponentCategory`) and an optional `type` (`oneof BatteryType, EVChargerType, InverterType`).
+* The `TargetComponents` message now accepts an optional `type` too. `.components.component_categories` is now deprecated. Instead `.components.component_categories_types`, a new `CategoryAndType` message that has a required `category` (`ComponentCategory`) and an optional `type` (`oneof BatteryType, EVChargerType, InverterType`) should be used.
```

Comment:
Maybe add a `## Deprecations` section? You don't really **need** to do anything to upgrade.

Reply:
I would just have completely moved this to the deprecations:

> `.components.component_categories` is now deprecated. Instead `.components.component_categories_types`, a new `CategoryAndType` message that has a required `category` (`ComponentCategory`) and an optional `type` (`oneof BatteryType, EVChargerType, InverterType`) should be used.